### ISSUE TITLE
Error: swarm has been destroyed

### ISSUE THREAD

**Issue by jameslibscie** [p2pcommons/sdk-js#89]
When using @hypergraph-xyz/cli, I get this error:
```
✔ Profile created! Now creating content
√ Title ... new parent test
√ Description ...
√ Select subtype » Design
√ License: https://creativecommons.org/publicdomain/zero/1.0/legalcode ... yes
dat://7108bf4d97bd3edb8cb61247d7948a00b863ae86b5a13b18784d43a056f5c25f
- Synchronizing network✖ Error: swarm has been destroyed
    at Swarm.flush (D:\Websites\hypergraph-xyz\cli\node_modules\hyperswarm\swarm.js:216:31)
    at D:\Websites\hypergraph-xyz\cli\node_modules\corestore-swarm-networking\index.js:113:30
    at Topic._stopDht (D:\Websites\hypergraph-xyz\cli\node_modules\@hyperswarm\discovery\index.js:123:29)
    at Topic.destroy (D:\Websites\hypergraph-xyz\cli\node_modules\@hyperswarm\discovery\index.js:71:10)
    at Discovery.destroy (D:\Websites\hypergraph-xyz\cli\node_modules\@hyperswarm\discovery\index.js:315:15)
    at NetworkResource._close (D:\Websites\hypergraph-xyz\cli\node_modules\@hyperswarm\network\index.js:175:20)
    at NetworkResource.Nanoresource.close (D:\Websites\hypergraph-xyz\cli\node_modules\nanoresource\index.js:104:8)
    at NetworkResource.Nanoresource.close (D:\Websites\hypergraph-xyz\cli\node_modules\nanoresource\index.js:77:54)
    at Swarm.destroy (D:\Websites\hypergraph-xyz\cli\node_modules\hyperswarm\swarm.js:288:18)
    at D:\Websites\hypergraph-xyz\cli\node_modules\corestore-swarm-networking\index.js:147:19
```

This happens upon:
- Creating a content module (the module is created successfully with `dat.json` file)
- Updating a content module's metadata
- Updating a profile module
- Opening a content folder (sometimes this works, sometimes this doesn't)
- Reading metadata (sometimes this works, sometimes this doesn't)

Using `0.5.0` of the SDK on Windows 10 in Windows Powershell (Admin)

**Comment by dpaez** [comment]
related to https://github.com/andrewosh/corestore-swarm-networking/pull/2

**Comment by dpaez** [comment]
temporarily fixed with #91
